dashboard/app/views/home/_recent_courses.html.haml

```diff
@@ -0,0 +1,20 @@
+:ruby
```

Comment:
@Bjvanminnen commented on one of my PRs that the logic for this should probably live in the model or controller and have unit tests. Seems like that notion would cross apply here?